src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  width: 198px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  font-weight: 400;

```

Comment:
it's default value - can be removed

Reply:
Not fixed 
Remove `font-weight: 400;` - it's default value 
Fix it everywhere